makefile

```diff
@@ -1533,7 +1533,7 @@ generate: \
 
 $(GENDIR)/includes/SDL2:
 	-$(call MKDIR,$@)
-	-$(call COPY,3rdparty/SDL2/include/,$(GENDIR)/includes/SDL2)
+	-$(call COPY,3rdparty/SDL2/include/.,$(GENDIR)/includes/SDL2)
```

Comment:
Absolutely works. Removing slash is much better in my test. I will change it. Thank you.